### ISSUE TITLE
NVDA Screen reader

### ISSUE THREAD

**Issue by ervinpm** [bassjobsen/Bootstrap-3-Typeahead#245]
Screen reader does not read type ahead values. Might need to add the following in the next and prev functions:

for the next function:

```
       // added for screen reader
       this.$element.val(next.text());
```

for the prev function:

```
      // added for screen reader
      this.$element.val(prev.text());
```
